modules/ding_library/ding_library.views_default.inc

```diff
@@ -135,6 +135,28 @@ function ding_library_views_default_views() {
   $handler->display->display_options['sorts']['title']['id'] = 'title';
   $handler->display->display_options['sorts']['title']['table'] = 'node';
   $handler->display->display_options['sorts']['title']['field'] = 'title';
+  /* Contextual filter: Content: Has taxonomy term ID */
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-937

Reply:
What does this change do exactly?